mkosi/__init__.py

```diff
@@ -1846,20 +1847,20 @@ def identify_cpu(root: Path) -> tuple[Optional[Path], Optional[Path]]:
     return (Path(f"{vendor_id}.bin"), None)
 
 
-def build_microcode_initrd(context: Context) -> Optional[Path]:
+def build_microcode_initrd(context: Context) -> list[Path]:
```

Comment:
Does the return type really need to be changed here? This function returns at most one item, and the previous signature was a better match for that.

Reply:
No, but then we need to mangle it in the caller, which has to return `list[Path]` since it might have to return multiple. @behrmann suggested changing it here instead, above:

> I think i would make sense to have build_microcode_initrd return a list[Path] now instead of an Optional[Path]. Even though it only ever returns an empty list or one with one element, it would make the whole thing more uniform.

No strong opinion, happy to do it either way